gdb-server/src/parser/mod.rs

```diff
@@ -127,7 +127,9 @@ pub enum BreakpointType {
 }
 
 pub fn parse_packet(input: &[u8]) -> Result<Packet> {
-    let parse_result = dbg_dmp(
+    // dbg_dmp takes Fn but everything in nom is now FnMut ?
```

Comment:
`dbg_dmp` is just debugging, so please just remove the commented out lines here.

Reply:
Sure, I was not sure if I did not see a way to make this work but to me it seems like an oversight in the 6.0 release maybe so removed it for now.